### ISSUE TITLE
Is it possible to draw an area on the map we are using now?

### ISSUE THREAD

**Issue by nandiheath** [g0vhk-io/HKAddressParser#34]
As now we have the constituency area info,
it would be nice if we draw it on the map for the single search result.

**Comment by cswbrian** [comment]
yes, we have got the geojson file, I am sure that google map API or leaflet can draw the polygon, but not sure for arcGIS which we are using now.

**Comment by ylchan87** [comment]
#47 should addressed this
